addons/hr/data/hr_data.xml

```diff
@@ -74,5 +74,25 @@
             <field name="value">True</field>
         </record>
 
+        <!-- Departure Reasons -->
+        <record id="departure_fired" model="hr.departure.reason">
+            <field name="sequence">0</field>
+            <field name="name">Fired</field>
+        </record>
+
+        <record id="departure_resigned" model="hr.departure.reason">
+            <field name="sequence">1</field>
+            <field name="name">Resigned</field>
+        </record>
+
+        <record id="departure_retired" model="hr.departure.reason">
+            <field name="sequence">2</field>
+            <field name="name">Retired</field>
+        </record>
+
+        <record id="departure_freelance" model="hr.departure.reason">
+            <field name="sequence">3</field>
+            <field name="name">Became Freelance</field>
+        </record>
```

Comment:
I propose to add `Became Butcher`.

Joke aside, this departure reason seems too specific to IT. 
Plus, I don't see why we need to track the new occupation of old employees.

A more relevant departure reason would be `Death`.

Reply:
It's needed for the "Bilan Social", but it's indeed specific for Belgium... I'll move to the Belgian payroll instead.

Thanks :-)

Reply:
FYI, it's not to "track the new occupation of old employees", it's for employees who stopped their classic contract to start working as a freelance in the same company.

Reply:
It makes sense then.